test/e2e/flowaggregator_test.go

```diff
@@ -532,6 +504,102 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false, checkBandwidth)
 		}
 	})
+
+	// testFlowAggregatorAntctl ensures antctl is accessible in a Flow Aggregator Pod
+	// and checks the output of antctl commands.
+	t.Run("testFlowAggregatorAntctl", func(t *testing.T) {
+		skipIfNotRequired(t, "mode-irrelevant")
+		flowAggPod, err := data.getFlowAggregator()
+		if err != nil {
+			t.Fatalf("Error when getting flow-aggregator Pod name: %v", err)
+		}
+		podName := flowAggPod.Name
+		for _, args := range antctl.CommandList.GetDebugCommands(runtime.ModeFlowAggregator) {
+			command := []string{}
+			if testOptions.enableCoverage {
+				antctlCovArgs := antctlCoverageArgs("antctl-coverage")
+				command = append(antctlCovArgs, args...)
+			} else {
+				command = append([]string{"antctl", "-v"}, args...)
+			}
+			t.Logf("command: %s", command)
+
+			t.Run(strings.Join(command, " "), func(t *testing.T) {
```

Comment:
I think we should keep the Run name short and precise like `Version` and `LogLevel` instead of keeping the complete command (duplicate to log in previous line)

Reply:
Current log and Run names are intended to keep consistence with the antctl tests for agent and controller. Do you think it would be better to provide a different but precise format?

Reply:
I see. In that case we can follow agent and controller conventions. Thanks!